### ISSUE TITLE
Seo images don't update mtime when asset settings are changed.

### ISSUE THREAD

**Issue by mlathrom** [khalwat/seomatic#404]
### Describe the bug
The mtime param appended to seo images doesn't take into account changes to asset settings local focal point.

The mtime param is created in line 156 of src/helpers/imageTransform.php. It checks the "dateModified" on the asset. The problem is that "dateModified" only takes into account changes to the actual asset file, not the asset settings. "dateUpdated" checks the asset settings.

### To reproduce
Steps to reproduce the behaviour:
1. Navigate to asset used as an seo image
2. Change the focal point of that image
3. Navigate to the page that asset appears in the head of
4. Check mtime

### Expected behaviour
Asset focal point should update in seomatic image meta tags.

### Versions
- Plugin version: 1.1.56
- Craft version: 4.4.7.1

**Comment by mlathrom** [comment]
I've tested a couple of solutions for this.

**Remove mtime param**
This fixed the issue with assets seomatic images updating when focal point is changed, but I'm not sure if the mtime param was added to solve some other issue.

**Check dateModified and dateUpdated and use the newest**
This will ensure that changes to the asset file and asset settings are taken into account when adding the mtime param.
